graphbolt/src/utils.h

```diff
@@ -9,6 +9,8 @@
 
 #include <torch/script.h>
 
+#include <optional>
```

Comment:
Can you place a comment on which torch versions are missing the include?

Reply:
Since we use std::optional directly in the code below, I guess such comment is not needed.